tests/Behat/UserGraphQLContext/UserGraphQLContext.php

```
@@ -267,31 +265,315 @@
+        $this->validateUserDataExists($decoded);
+    }
+
+    /**
+     * @param array<string, string|int|float|bool|null> $decoded
+     */
+    private function validateNoErrors(array $decoded): void
+    {
+        if (isset($decoded['errors'])) {
+            $errorMessage = 'GraphQL Errors: '
+                . json_encode($decoded['errors']);
+            throw new \Exception($errorMessage);
```

PHPMD (2.15.0)

Missing class import via use statement (line '387', column '23'). (Clean Code Rules)

(MissingImport)